docs/site/Memory-connector.md

```diff
@@ -22,9 +22,6 @@ The memory connector supports:
 
 - Standard query and create, read, update, and delete operations, so you can
   test models against an in-memory data source.
-- Geo-filtering when using the `find()` operation with an attached model.
-  See [GeoPoint class](http://apidocs.loopback.io/loopback-datasource-juggler/#geopoint) for
-  more information on geo-filtering.
```

Comment:
Instead of removing geo-filtering from the docs, can we please add a note that GeoPoint type is not supported by LB4 yet and link readers to the GH issue tracking GeoPoint support?

To make this future proof, I would say something like "while the memory connector supports location-based queries and filtering, LoopBack 4 does not support GeoPoint type yet. See GitHub issue loopback-next#xyz for more details".

That way the note will stay useful even after GeoPoint support is implemented.

Reply:
Taking from https://github.com/strongloop/loopback-next/pull/6319#discussion_r486712866;

Since GeoPoint is a repository-level concept that applies to multiple connectors, I think we should not be putting these kinds of info in a connector's documentation.

If need be, we should be putting it in a more general docs (e.g. Repository, Model, LoopBack Types) that is more likely to be seen by the target users - those who want to know about GeoPoint as a general concept.

AFAIK, the memory connector does not have any specific implementation quirks with GeoPoint. Hence, it would be better to shift GeoPoint into one of the aforementioned docs.

Reply:
Fair enough. We can iterate and improve this part later, if needed at all.